assets/methods_description_template.yml

```diff
@@ -3,8 +3,6 @@ description: "Suggested text and references to use when describing pipeline usag
 section_name: "nf-core/sarek Methods Description"
 section_href: "https://github.com/nf-core/sarek"
 plot_type: "html"
-## TODO nf-core: Update the HTML below to your prefered methods description, e.g. add publication citation for this pipeline
```

Comment:
shouldn't we add some description on sarek here?

Reply:
I looked at the other pipelines, and nothing is there, the only difference I saw in some pipelines was the comments removed.
I'm not sure why it's in the TEMPLATE